### PR TITLE
Remove feature toggle, update styling

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: components/record/MediaThumbnailGrid.vue [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
       :class="{ 'selected' : isSelected(thumbnail) }"
+      :style="{ 'display' : index > 10 && !showAll ? 'none' : 'inline' }"
       :data-about="thumbnail.about"
       :data-qa="`media thumbnail #${index + 1}`"
       thumbnail
       alt=""
+      class="mb-2 mr-2 rounded-0"
       @click="clickThumbnail(thumbnail.about)"
     />
+    <button
```
Comment (rwd) [inline components/record/MediaThumbnailGrid.vue]: This needs to load 11 more, not all the rest. That may imply that there would be both a "show more" and "show less" button visible at the same time.

See for instance record /9200355/BibliographicResource_3000117729351 with 159 thumbnails.

Comment (rwd) [reply components/record/MediaThumbnailGrid.vue]: ... I thought, but the acceptance criteria are in fact ambiguous.

File: components/record/MediaThumbnailGrid.vue [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
     data-qa="media thumbnail grid"
+    class="d-flex flex-wrap mb-3"
   >
     <!-- TODO: populate alt, but with what? -->
     <b-img
       v-for="(thumbnail, index) of thumbnails"
       :key="index"
       :src="thumbnail.src"
       :class="{ 'selected' : isSelected(thumbnail) }"
+      :style="{ 'display' : index > 10 && !showAll ? 'none' : 'inline' }"
```
Comment (monkemedia) [inline components/record/MediaThumbnailGrid.vue]: Probably be cleaner to add a class to hide the image `.d-none` https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements

File: components/record/MediaThumbnailGrid.vue [hunk 3]
```diff
@@ -1,19 +1,29 @@
...
       :data-qa="`media thumbnail #${index + 1}`"
       thumbnail
       alt=""
+      class="mb-2 mr-2 rounded-0"
       @click="clickThumbnail(thumbnail.about)"
     />
+    <button
+      v-if="thumbnails.length > 10"
+      class="pb-0"
+      @click="toggleThumbnails()"
```
Comment (monkemedia) [inline components/record/MediaThumbnailGrid.vue]: remove `()`

**Pull request merged**